apps/mobile/src/features/send/components/error-state.tsx

```diff
@@ -0,0 +1,27 @@
+import { t } from '@lingui/macro';
+import { Image } from 'expo-image';
+
+import { ArrowRotateClockwiseIcon, Box, Button, Text } from '@leather.io/ui/native';
+
+interface ErrorStateProps {
+  onRetry: () => void;
+}
+
+export function ErrorState({ onRetry }: ErrorStateProps) {
+  return (
+    <Box alignItems="center" justifyContent="center" gap="4" style={{ paddingTop: 64 }}>
+      <Image
```

Comment:
Maybe this can be a `prop` and we can re-use it? 

I can do that in [LEA-2562](https://linear.app/leather-io/issue/LEA-2562/error-boundary)

Reply:
Just the `<Image` so we can pass in different ones